### PR TITLE
add example pattern for documentation

### PR DESCRIPTION
**Pull request by precious-onyenaucheya-ons**
### What is the context of this PR?

This PR addresses #2831 and #2832. I have modified the error status pages  so that  links  appears at the end  of text, in order to comply with the style guide.

I have also added a pattern for example survey introduction.
### How to review this PR

Check that links in the error status pages appear at the end and that there is an example for survey introduction.
### Checklist

This needs to be completed by the person raising the PR.

- [x] I have selected the correct Project for this PR (Design System) and selected the correct status
- [x] I have selected the correct Assignee
- [x] I have linked the correct Issue

### CODE REVIEW

**Review by rmccar — CHANGES_REQUESTED** [review]
Couple of typos and I would ask Oscar if the new survey section introduction page should be under the interstitial pages category. I would've thought not or if they are under the same category the category should be renamed.

File: src/patterns/error-status-pages/example-page-not-found-error-with-code.njk [hunk 1]
```diff
@@ -17,7 +17,7 @@ layout: ~
     <h1>Page not found</h1>
     <p>If you entered a web address, check it is correct.</p>
     <p>If you pasted the web address, check you copied the whole address.</p>
-    <p>If the web address is correct or you selected a link or button, <a href="#0">contact us</a> for more help.</p>
+    <p>If the web address is correct or you selected a link or button, <a href="#0">contact us</a>.</p>
```
Comment (rmccar) [inline src/patterns/error-status-pages/example-page-not-found-error-with-code.njk:20]:
```suggestion
    <p>If the web address is correct, or you selected a link or button, <a href="#0">contact us</a>.</p>
```

File: src/patterns/error-status-pages/example-page-not-found-error.njk [hunk 2]
```diff
@@ -14,5 +14,5 @@ layout: ~
     <h1>Page not found</h1>
     <p>If you entered a web address, check it is correct.</p>
     <p>If you pasted the web address, check you copied the whole address.</p>
-    <p>If the web address is correct or you selected a link or button, <a href="#0">contact us</a> for more help.</p>
+    <p>If the web address is correct or you selected a link or button, <a href="#0">contact us</a>.</p>
```
Comment (rmccar) [inline src/patterns/error-status-pages/example-page-not-found-error.njk:17]:
```suggestion
    <p>If the web address is correct, or you selected a link or button, <a href="#0">contact us</a>.</p>
```

**Comment by rmccar** [comment]
There also seems to be another new example that needs to be added if you look at the bit at the bottom of #2832

**Comment by precious-onyenaucheya-ons** [comment]
> There also seems to be another new example that needs to be added if you look at the bit at the bottom of #2832

I have added a comment in the ticket to ignore that request

**Comment by rmccar** [comment]
> > There also seems to be another new example that needs to be added if you look at the bit at the bottom of #2832
> 
> I have added a comment in the ticket to ignore that request

I would check with Oscar that, he's happy with that example you have mentioned and if he is, if the content of that example should change

**Review by rmccar — APPROVED** [review]

**Review by alessioventuriniAND — APPROVED** [review]

**Pull request merged**